mava/components/jax/updating/parameter_server.py

```diff
@@ -148,20 +136,14 @@ def on_parameter_server_init_start(self, server: SystemParameterServer) -> None:
                     net_type_key
                 ][agent_net_key].params
 
-        # Create the checkpointer
-        if self.config.checkpoint:
-            server.store.last_checkpoint_time = 0
-
-            # Only save variables that are not empty.
-            save_variables = {}
-            for key in server.store.parameters.keys():
-                var = server.store.parameters[key]
-                # Don't store empty tuple (e.g. empty observation_network) variables
-                if not (type(var) == tuple and len(var) == 0):
-                    save_variables[key] = var
-            server.store.system_checkpointer = savers.Checkpointer(
-                save_variables, self.config.experiment_path, time_delta_minutes=0
-            )
+        # Only save variables that are not empty.
+        server.store.saveable_parameters = {}
```

Comment:
This variable could be removed. I have kept it as this is what we do in TF with obs networks

Reply:
Actually I decided to remove it